### ISSUE TITLE
eventsource continues launching more containers if left open

### ISSUE THREAD

**Issue by rgbkrk** [jupyterhub/binderhub#317]
If you leave the eventsource open after "ready" it continues launching more

Reproduction steps:

* Open about:blank
* Open DevTools Console
* Paste:

```
es = new window.EventSource('https://mybinder.org/build/gh/nteract/vdom/master');
es.onmessage = msg => console.log(msg)
```

* wait...

-----

Not a crazy deal for me, I just happened to notice while hitting the event source API directly. My current code does a `.close` appropriately. Figured this should probably be cleaned up though.

**Comment by choldgraf** [comment]
I think we just got burned by this with the nbinteract repository. @rgbkrk did you figure out a way to work around this from your side of things?

**Comment by rgbkrk** [comment]
Yeah, we close the event source after we get a binder. It wasn't an issue at the time, it just smelled like a likely operational problem waiting to happen.

**Comment by choldgraf** [comment]
> it just smelled like a likely operational problem waiting to happen.


![](https://media.giphy.com/media/ap6wcjRyi8HoA/giphy.gif)

**Comment by manics** [comment]
Is this still a problem?
